frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -130,6 +142,8 @@ const Metric = (props: MetricProps) => {
         props.type,
         value,
         showValue,
+        deltaColor,
```

Comment:
Isn't there are way *not* to indicate these colors if the user doesn't change them after all?
That would allow us to not go and find the Plotly's default values, but also to follow their changes when they occur...

Reply:
If users don't specify the colors or "invert", the code takes them as undefined so the delta will be displayed in the default color.

Reply:
Yes yes I understand.
My question really was about why we need to inject any color value in this code.
Pos and neg default color values should be left to Plotly to decide. See my point? No big deal by the way.
Great job!